docs/rules/no-store-async.md

```diff
@@ -0,0 +1,57 @@
+---
+pageClass: "rule-details"
+sidebarDepth: 0
+title: "svelte/no-store-async"
+description: "disallow using async/await inside svelte stores"
+since: "v3.1.0"
```

Comment:
We need to update here when we decide the release version of this rule.

Reply:
We don't need it now as it is automatically added by the release script.

```suggestion
```

Reply:
removed!
https://github.com/ota-meshi/eslint-plugin-svelte/pull/225/commits/c05ef7cc01b234f3627bd4f2209214169884d474